1.9/usage/tutorials/dcos-101/resources.md

```diff
@@ -0,0 +1,98 @@
+---
+post_title: Understanding Resources
+nav_title: Understanding Resources
+menu_order: 8
+---
+
+# Prerequisites
+* A [running DC/OS cluster](/docs/1.9/usage/tutorials/dcos-101/cli/) with [the DC/OS CLI installed](/docs/1.9/usage/tutorials/dcos-101/cli/).
+* [app2](/docs/1.9/usage/tutorials/dcos-101/app2/) deployed and running in your cluster.
+
+# Objective
+In this section we will learn how to monitor and understand our resource utilization, how resource limits are enforced, and how to debug resource management issues.
```

Comment:
`In this section you will learn...` In general you want to address the readers in the first person - not as `we`.